src/App.jsx

```diff
@@ -25,7 +26,23 @@ export const alex = {
 
 export const App = () => (
   <div className="App">
-    <section className="Person">
+    <Person
+      name={misha.name}
+      age={misha.age}
+      partner={misha.partnerName}
+      sex={misha.sex}
```

Comment:
do not pass separate properties
you should pass an entire person object as a prop